core/test/unit/middleware/check-ssl_spec.js

```diff
@@ -41,6 +41,29 @@ describe('checkSSL', function () {
         done();
     });
 
+    it('should redirect to subdirectory (frontend)', function (done) {
+        req.originalUrl = '/blog';
+        // req.url can be different than req.originalUrl when mounting or url
+        // rewriting happens.
+        req.url = '/';
+        res.isAdmin = true;
```

Comment:
Hey @SeanHayes Thanks for you help improving our tests :) Very appreciating that!

I think `res.isAdmin` in this case does not make sense? If i understand this right, you are trying to test if a http request to a non admin url get's redirected to https, correct?
`res.isAdmin` is only true, when we are hitting the admin routes - see and follow https://github.com/TryGhost/Ghost/blob/master/core/server/app.js#L67